package.json

```
@@ -17,7 +17,11 @@
         "test": "yarn test:pinia --run && yarn test:e2e",
         "test:pinia": "nuxi prepare && nuxi generate && vitest",
         "test:pinia:coverage": "vitest run --coverage",
-        "test:e2e": "cypress run --e2e --browser chrome --record --key 04a1fe54-0011-407b-b7d7-fcb9b2daf1e6"
+        "test:e2e": "NODE_ENV=test dotenv -e .env.test -o -- yarn test:e2e:start",
+        "test:e2e:start": "yarn test:env:echo && cypress run --e2e",
```

someone of a personal preference here, but I think we should avoid "subscripts" where possible for a few reasons:
-  While the reusability and separation is nice, there's no way to look at the list of scripts and know which ones should be run vs not.
- Too many scripts. When someone is new to the project, it can be intimidating. Especially when there's several that look almost the same. `e2e:start` vs `e2e` vs `e2e:open` for example
- Ideally (where possible) there should only be one script to start something. So basically I would combine `test:e2e` + `test:e2e:start` + `test:e2e:open` + `test:e2e:open:start`+`test:env:echo` all into a single `test:e2e` script. I get it can be long, but I think that's ok. It only matters when someone is trying to edit the script.

---

@ermish 

What do you think this way?:

```suggestion
        "test:e2e": "NODE_ENV=test dotenv -e .env.test -o -- bash -c 'echo 🧪 $NODE_ENV environment loaded 🧪 && cypress run --e2e'",
        "test:e2e:open": "NODE_ENV=test dotenv -e .env.test -o -- bash -c 'echo 🧪 $NODE_ENV environment loaded 🧪 && cypress open'"
```

---

after talking in slack, we've updated the commands